server/graphql/v2/object/Order.js

```diff
@@ -221,6 +221,20 @@ export const Order = new GraphQLObjectType({
           }
         },
       },
+      memo: {
+        type: GraphQLString,
+        description:
+          'Memo field which adds additional details about the order. For example in added funds this can be a note to mark what method (cheque, money order) the funds were received.',
+        async resolve(order, _, { remoteUser }) {
+          const collective = await models.Collective.findOne({ where: { id: order.CollectiveId } });
```

Comment:
I would even do something like:

```
order.collective = order.collective || (await loaders.Collective.byId.load(order.CollectiveId));
```

Reply:
Why fetch if it's already available in the order object? And still be defensive so we don't have to think if it's available or not.

Reply:
Thanks. This makes more sense. I've refactored this part. 👍🏽